### DISCUSSION TITLE
[Feature Request] Different sort option for main views (notes, archived, files, etc.)

### DISCUSSION THREAD

**Discussion by MustachedMario** [standardnotes/forum#2862] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
I sort my notes to "title a-z", but if i look into the archive or files view, i often want a different sort order for faster access. e.g. "title z-a" or "date created" in archive.

**Describe the solution you'd like**
Provide an option to save a sort order per main view.

**Describe alternatives you've considered**
More Smart View options to manipulate sort order.

**Comment by myreli** [comment]
This is now available, as well as many other settings per view: 

<img width="653" alt="image" src="https://user-images.githubusercontent.com/17554234/221365691-18b45418-bc2a-43fb-a845-f11106b6dc8d.png">
